kwok/cloudprovider/cloudprovider.go

```diff
@@ -133,49 +132,37 @@ func (c CloudProvider) toNode(nodeClaim *v1beta1.NodeClaim) (*v1.Node, error) {
 	//nolint
 	newName = fmt.Sprintf("%s-%d", newName, rand.Uint32())
 
-	capacityType := v1beta1.CapacityTypeOnDemand
-	requirements := scheduling.NewNodeSelectorRequirementsWithMinValues(nodeClaim.
-		Spec.Requirements...)
-	if requirements.Get(v1beta1.CapacityTypeLabelKey).Has(v1beta1.CapacityTypeSpot) {
-		capacityType = v1beta1.CapacityTypeSpot
-	}
+	requirements := scheduling.NewNodeSelectorRequirementsWithMinValues(nodeClaim.Spec.Requirements...)
 	req, found := lo.Find(nodeClaim.Spec.Requirements, func(req v1beta1.NodeSelectorRequirementWithMinValues) bool {
 		return req.Key == v1.LabelInstanceTypeStable
 	})
 	if !found {
 		return nil, fmt.Errorf("instance type requirement not found")
 	}
 
-	minInstanceTypePrice := math.MaxFloat64
 	var instanceType *cloudprovider.InstanceType
+	var bestOffering *cloudprovider.Offering
```

Comment:
```suggestion
	var cheapestOffering *cloudprovider.Offering
```

Is it "best" or "cheapest"? I could see this being "best" if we were working off something different from price, but since we are using price as an optimization proxy here, it seems like "cheapest" might be the more operative word here